src/shared/components/query/GenesetsVolcanoSelector.tsx

```diff
@@ -87,7 +88,7 @@ export default class GenesetsVolcanoSelector extends QueryStoreComponent<Geneset
                     && (this.props.plotData) && (this.props.maxY)
                     && (this.store.volcanoPlotTableData.isComplete) && (this.props.data) && (
                 <VictoryChart
-                    theme={VictoryTheme.material}
+                    theme={CBIOPORTAL_VICTORY_THEME}
```

Comment:
@oplantalech we have our own Victory theme.  It's a copy of material which we have altered in a few ways.  Also, it's only necessary on the outer Chart component.